package-configuration/linux-cli/get-cloudify.py

```diff
@@ -119,14 +120,17 @@ def run(cmd, sudo=False):
         prn('Executing: {0}...'.format(cmd))
     proc = subprocess.Popen(
         cmd, shell=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+    stdout = ''
     # while the process is still running, print output
     while proc.poll() is None:
         if VERBOSE:
-            _print_proc_out(proc)
-    _print_proc_out(proc)
+            stdout += _print_proc_out(proc)
+    stdout += _print_proc_out(proc)
     stderr = proc.stderr.read()
     if len(stderr) > 0:
         prn('STDERR: {0}'.format(stderr))
+    proc.stdout = stdout
+    proc.stderr = stderr
```

Comment:
this is kinda bad, Popen.stdout/stderr is expected to be one of two things: None, or file object. This makes it string. Maybe just convert run to return tuple (exitcode, stdout, stderr) instead?